CHANGELOG.md

```diff
@@ -590,7 +590,7 @@ All notable changes to this project will be documented in this file. See [standa
 ### Docs
 - fix function typos
 - setting up remote access to kind clusters
-- wireguard configuraiton for OS X
+- wireguard configuration for OS X
```

Comment:
I wonder what's the deal with wireguard. What's is doing here?

Reply:
I mean it's unrelated to this review. Still...

Reply:
@lkingland any idea what it was there?

Reply:
This was related to an old example which is no longer used